src/main/kotlin/BaseTurn.kt

```diff
@@ -0,0 +1,2 @@
+class BaseTurn(val playerColor: Color, val from: Pair<Int, Int>, val to: Pair<Int, Int>) {
```

Comment:
1) Empty `{}` may be removed
2) Read about data classes, maybe that's one you need here. Especially if you want to print instances or serialize them